### PR TITLE
Test utilities for REST APIs

### PR DESCRIPTION
**Pull request by igaray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,28 @@ erlang katana
 
 ![samuari](https://raw.githubusercontent.com/unbalancedparentheses/katana/master/images/samurai.jpg)
 
-Even if you love Erlang as I do, from time to time you might ask yourself why some functions you normally find in other languages are not part of the erlang's standard library. When you ask yourself that type of question you should remember that an estimated 2 million people are currently working in COBOL and 1.5 million new lines of COBOL code are written every day. After feeling bad for those developers, you should send a pull request to erlang katana with the functions you use on a daily basis.
+Even if you love Erlang as I do, from time to time you might ask yourself why
```
Comment (elbrujohalcon) [inline README.md]: "I" -> "we" ;)

**Comment by elbrujohalcon** [comment]
@igaray maybe the functions (or the module) should have less generic names (i.e. including the word `rest` somewhere)

**Comment by igaray** [comment]
Updated

File: src/ktn_test_utils.erl [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+-export(
+  [ assert_response/4
+  , test_response/4
+  ]).
+
+-type test_subject() :: status | headers | body.
+-type match_type()   :: exact | partial.
+
+-spec assert_response(test_subject(), match_type(), map(), chtc:response()) -> ok.
```
Comment (gadgetci) [inline src/ktn_test_utils.erl]: According to **Elvis**:
> Line 11 is too long: -spec assert_response(test_subject(), match_type(), map(), chtc:response()) -> ok..

**Pull request merged**